.github/workflows/ci.yaml

```diff
@@ -111,6 +111,36 @@ jobs:
         with:
           arguments: ':schema-loader:dockerfileLint'
 
+  build-check-example-project:
+    name: Build check for 'Getting Started' example project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up JDK ${{ env.JAVA_VERSION }} (${{ env.JAVA_VENDOR }})
+        uses: actions/setup-java@v4
+        with:
+          java-version: ${{ env.JAVA_VERSION }}
+          distribution: ${{ env.JAVA_VENDOR }}
+
+      - name: Build Getting Started project
+        run: cd docs/getting-started && ./gradlew assemble
```

Comment:
To speed up the build by caching the gradle build output, I think we could use the https://github.com/gradle/actions/tree/main/setup-gradle 
What do you think ?

Reply:
Good point! [Revised the PR to use it](https://github.com/scalar-labs/scalardb/pull/1871/commits/feaf42f1bae99820b4b2ed20f43b1c2eb029761d).

Reply:
Thanks.
I checked the CI to see what was cached after the build, and I was surprised to see that nothing was! 
I found out that, by default, the cache is only created by workflow running on the default branch to avoid overloading the cache storage (max 10GB & retention of 7 days), and non-default branch only read the cache but never write to it. Even though the CI is not triggered automatically on the default branch anymore, we still have the `release-snapshot` and `vulnerability check` workflow populating the cache that can be use by the CI workflow.

cf. https://github.com/gradle/actions/blob/main/docs/setup-gradle.md#properties-of-the-github-actions-cache